readme.md

```diff
@@ -1,145 +1,151 @@
+
+## Core Learning Objective
+
+*	Use NodeJS APIs to interact with files and the web
+
+## Sub-Objectives
+
+* Define CRUD
+* Read from files
+* Overwrite files
+* Programmatically update files
+
+### Prereqs
+
+Before starting this lesson, Fork & Clone this repository. Make sure that you can run `npm start` and that you get the following output:
+```
+Hello, Node!
+```
+
+### Instructions & Guiding Questions
+
+- [ ] Begin by renaming `data/pets.sample.json` to `pets.json`
+
+* **Question:** In this lesson we will be modifying the contents of the `pets.json` file. Why do you think that file appears inside of the `.gitignore` file?
+
+* **Your Answer:**
+
+--- Files referenced in `.gitignore` are ignored during commit. A file or directory can be identified in `.gitignore` for many reasons, such as sensitive data files, api keys, etc. Let's pretend that pets.json is a sensitive data file.
+
+- [ ] Take a look at the NodeJS documentation for the [fs module](https://nodejs.org/api/fs.html). When looking at documentation like this it can be overwhelming to start but you likely know more than you think. Take a moment to find a few concepts you understand.
+
+---
+
+- [ ] One of the most common patterns we'll come across as web developers is the concept of [CRUD](https://www.codecademy.com/articles/what-is-crud). Take a moment to define what CRUD represents.
+
+* **Question:** Imagine you have a file called `classmates.txt`. For each part of CRUD, describe how the action would interact with the file.
+
+* **Your Answer:**
+
+--- CRUD = Create/Read/Update/Delete. Create describes the function(s) used to create the classmates.txt file. Read describes the function(s) used to read the contents of the classmates.txt file. Update describes the function(s) used to update the contents of the classmates.txt file. Delete describes the function(s) used to delete contents from the classmates.txt file.
+
+- [ ] Consider the above and then look back through the [fs module](https://nodejs.org/api/fs.html) documentation.
+
+* **Question:** What methods represent each CRUD action?
+
+* **Your Answer:**
+
+---
+-Create: `fs.writeFile(); fs.writeFileSync()`
+-Read: `fs.readFile(); fs.readFileSync()`
+-Update: `fs.writeFile(); fs.writeFileSync(); fs.appendFile(); fs.appendFileSync()`
+-Remove: `fs.ftruncate(); fs.ftruncateSync(); fs.unlink(); fs.unlinkSync()`
+
+- [ ] Take a look at the following two methods: [fs.readFile()](https://nodejs.org/api/fs.html#fs_fs_readfile_path_options_callback) and [fs.readFileSync()](https://nodejs.org/api/fs.html#fs_fs_readfilesync_path_options).
+
+* **Question:** What is the difference between these two methods?
+
+* **Your Answer:**
+
+--- `fs.readfile()` is asynchronous, meaning it does not run in parallel to other functions/code and will block/must complete first. `fs.readFileSync()` is synchronous, meaning it can run independently of the rest of the code. `fs.readfileSync()` does not take a callback function.

```

Comment:
I think you have these definitions switched! Synchronous code will block, while async code will not.